ui/src/components/ui/tooltip.tsx

```diff
@@ -19,8 +21,8 @@ const TooltipContent = React.forwardRef<
         ref={ref}
         sideOffset={sideOffset}
         className={cn(
-            `z-50 overflow-hidden rounded-md border border-slate-200 bg-black px-3 py-1.5
-       text-sm text-white shadow-md animate-in fade-in-0 zoom-in-95
+            `z-50 overflow-hidden rounded-md border border-slate-200 bg-white px-3 py-1.5
+       text-sm text-slate-950 shadow-md animate-in fade-in-0 zoom-in-95
```

Comment:
Keep the tooltip as bg-black and text-white since most of the tooltips are those colors. If you are planning to use the tooltip with a white background and black text, change that component individually.